custom_components/econet300/sensor.py

```diff
@@ -163,8 +161,9 @@ def create_mixer_sensors(coordinator: EconetDataCoordinator, api: Econet300Api):
 
     for i in range(1, AVAILABLE_NUMBER_OF_MIXERS + 1):
         string_mix = str(i)
-        if string_mix in MIXER_MAP:
-            for key, value in MIXER_MAP.get(string_mix).items():
+        mixer_map_value = MIXER_MAP.get(string_mix)
```

Comment:
Good to be supplied with default value 

`mixer_map_value = MIXER_MAP.get(string_mix, None)`